draft-ietf-taps-interface.md

```diff
@@ -531,6 +531,12 @@ Transport Properties can have one of a set of data types:
   selection; see {{selection-props}}. When querying, a Preference is of
   type Boolean, with `true` indicating that the Selection Property
   has been applied.
+- Tuple: A specific grouping of multiple value types.
```

Comment:
```suggestion
- Tuple: An ordered grouping of multiple value types.
```
Unless "specific" really means something here? But I wouldn't know what it means.